src/hotspot/share/code/aotCodeCache.cpp

```diff
@@ -649,6 +649,12 @@ void AOTCodeCache::Config::record() {
   _contendedPaddingWidth = ContendedPaddingWidth;
   _objectAlignment       = ObjectAlignmentInBytes;
   _gc                    = (uint)Universe::heap()->kind();
+  _cpu_features_size     = (uint)VM_Version::cpu_features_size();
+  void* cpu_features_buffer = (void*)(this+1);
```

Comment:
I don't like. Store it in separate space and reference (offset in archive) it from Config.